src/App.jsx

```diff
@@ -30,6 +31,7 @@ const router = createBrowserRouter(
       <Route element={<OperatorList />} path="/operators" />
       <Route element={<OperatorDetails />} path="/operators/:address/:tab?" />
       <Route element={<LST />} path="/lst" />
+      <Route element={<SubscriptionPlans />} path="/subscriptions" />
```

Comment:
Making the route available for logged in users only:

```suggestion
      <Route element={<PrivateOutlet />} path="/subscriptions">
        <Route element={<SubscriptionPlans />} index />
      </Route>
```